tests/unit/manage/test_views.py

```diff
@@ -389,7 +391,14 @@ def test_change_primary_email(self, monkeypatch, db_request):
         )
         view = views.ManageAccountViews(db_request)
 
+        send_email = pretend.call_recorder(lambda *a: None)
+        monkeypatch.setattr(
+            views, 'send_primary_email_change_email', send_email
+        )
         assert view.change_primary_email() == view.default_response
+        assert send_email.calls == [
+            pretend.call(db_request, db_request.user, email.email)
```

Comment:
@di After we merged this PR, I created [another PR](https://github.com/pypa/warehouse/pull/3111) with the latest master - but I see my [Travis test failing](https://travis-ci.org/pypa/warehouse/jobs/350764085) at this line which is no way related to #3111 . I'm not able to replicate the test failure locally.  

Could you please help me with this? Thanks

Reply:
Strange! Now when I updated my branch there wasn't any test failure.

Reply:
See my comment here: https://github.com/pypa/warehouse/pull/3158#discussion_r173226379